architecture/index.md

```diff
@@ -0,0 +1,31 @@
+# Architecture overiew
+
+## How data flows through the CDapp
+
+In principle, we want to handle all interactions with the chain outside of the client. We do this using the block ingestor and lambda functions.
+
+### The Block Ingestor
+
+A very common pattern at Colony is taking some data from the user in the ui (usually via a form), and then using that data to call some method in the `colony-js` library, which in turn calls the respective method on the colony network contracts (written in solidity). We coordinate all of this via `redux` actions.
+
+At a glance:
+
+1. Use the ActionForm or ActionButton components to dispatch an action to the store
+2. That action will get picked up by the `redux-saga` middleware, and if there's a listener attached, will run the corresponding saga
+3. Inside the saga, we interact with our ColonyNetwork contracts using `colony-js`. We also dispatch actions to the store every time the state of a transaction changes, so the `GasStation` can display this information to the user.
```

Comment:
This needs to specifically mention Chain Transactions, as they are the import part in this equation. "how" we get them sent are implementation minutiae